src/java/picard/analysis/CollectAlignmentSummaryMetrics.java

```diff
@@ -60,6 +61,13 @@
  * <li>Strand balance - reads mapped to positive strand / total mapped reads</li>
  * </ul>
  * Metrics are written for the first read of a pair, the second read, and combined for the pair.
+ *
+ * Chimeras are identified if any of the of following criteria are met:
+ * <ul>
+ * <li>the insert size is larger than MAX_INSERT_SIZE</li>
+ * <li>the paired end orientation is different that the expected orientation</li>
```

Comment:
"than" not that.
Also, please add: <li>the reads map to different contigs
(I'm just copying from the if statement in AlignmentSummaryMetricsCollector)